src/models.py

```diff
@@ -0,0 +1,36 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+import auth_util
+
+db = SQLAlchemy()
+migrate = Migrate()
+
+class User(db.Model):
+    user_id = db.Column(db.Integer, primary_key=True)
```

Comment:
Done. The reason I didn't use id is because id is a built-in in python, but I think this is fine.